src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -172,7 +172,7 @@ def app_run(
 
     processor = NativeAppRunProcessor(
         project_definition=cli_context.project_definition,
-        project_root=cli_context.project_root,
+        project_root=cli_context.project_root.native_app,
```

Comment:
did you mean to update project_definition instead?

Reply:
Yes, thanks for catching this!